### ISSUE TITLE
Reports should include more details about main contributions

### ISSUE THREAD

**Issue by AlexDaniel** [JJ/TPF-Grant#3]
I was expecting a section like “most notable documentation tickets resolved”, or something like that. Basically, anything that would explain in more detail which major contributions were done, and not just graphs that show a lot of commits.

**Comment by AlexDaniel** [comment]
IRC discussion: https://irclog.perlgeek.de/perl6/2018-05-20#i_16186414

**Comment by JJ** [comment]
The whole point of the grant was to address issues... But I guess it's fair
to highlight some specific contributions, although I would be hard pressed
to select a few. Some contributions of a single commit have meant a lot of
work, while others with lots of commits weren't so difficult to solve.

**Comment by AlexDaniel** [comment]
Then at least a list of issues that were closed?

**Comment by JJ** [comment]
For instance, this was a lot of work.
https://github.com/perl6/doc/issues/2022 but it was not too significant in
the great scheme of things. This one was one or a couple of commits
https://github.com/perl6/doc/issues/732 but required some hard work and
many trips to StackOverflow... I guess both would qualify for that section,
right?

**Comment by JJ** [comment]
El dom., 20 may. 2018 a las 20:32, Aleks-Daniel Jakimenko-Aleksejev (<
notifications@github.com>) escribió:

> Then at least a list of issues that were closed?
>

There are around 150 so far, but I already download all the issues from
time to time and can generate that pretty easily...

**Comment by AlexDaniel** [comment]
I guess that only includes issues closed by you? :)

**Comment by JJ** [comment]
No, I download every issue from the beginning of the repo. I can tell you the exact quote if you want, but in April and May together I would say it's in the high 100s.

**Comment by AlexDaniel** [comment]
But that is exactly my point, I think. The report should probably say “here are the issues that I resolved/closed: … … … …”.

**Comment by AlexDaniel** [comment]
Probably with some additional comments, where necessary (e.g. “this is a big issue that took a lot of effort”, “here are other smaller issues:”, etc.).

**Comment by JJ** [comment]
Um, almost, but no cigar:
```utils [master] % grep closed ../data/issues.csv | grep JJ | grep 2018-04 | wc
     82     492    5895
utils [master] % grep closed ../data/issues.csv | grep JJ | grep 2018-05 | wc
     59     354    4307
```

If I include those closed before *officially* starting to work on the grant, yes...
```
utils [master] % grep closed ../data/issues.csv | grep JJ | grep 2018-0[345] | wc
    157     942   11342
```
But you are right, I should generate a markdown with closed issues and maybe add some comments to it. There are also some non-closed issues (like the intermitent error with the tests) that are not really closed and also could be mentioned...

**Comment by JJ** [comment]
The problem is also that some issues figure as "closed" but without closing date if they have been opened again... Anyway, there's something now to that effect.
